src/app/components/SearchResultsPage/SearchResultsPage.jsx

```diff
@@ -143,7 +135,7 @@ const SearchResultsPage = (props, context) => {
                     total={totalHits}
                     perPage={50}
                     page={parseInt(page, 10)}
-                    to={{ pathname: urlSearchString }}
```

Comment:
OK! So if there's no value of  `this.props.to` for `<Pagination>` we will use the value `{ pathname: '#' },` then the value of `<Link>`'s `to` will be `to={{ pathname: /search?{ pathname: '#' } }}`. I assume it should just be `to={{ pathname: # }}`?